sky/provision/docker_utils.py

```diff
@@ -330,7 +330,7 @@ def initialize(self) -> str:
             'mkdir -p ~/.ssh;'
             'cat /tmp/host_ssh_authorized_keys >> ~/.ssh/authorized_keys;'
             'sudo service ssh start;'
-            'sudo sed -i "s/mesg n/tty -s \&\& mesg n/" ~/.profile;'
+            'sudo sed -i "s/mesg n/tty -s \\&\\& mesg n/" ~/.profile;'
```

Comment:
Starting in Python 3.12, there is a warning `.../sky/provision/docker_utils.py:333: SyntaxWarning: invalid escape sequence '\&'.
This should be what's intended.